src/libNode/FinalBlockProcessing.cpp

```diff
@@ -721,6 +716,13 @@ bool Node::ProcessFinalBlockCore(const bytes& message, unsigned int offset,
     return false;
   }
 
+  auto resumeBlackList = []() mutable -> void {
+    this_thread::sleep_for(chrono::seconds(RESUME_BLACKLIST_DELAY_IN_SECONDS));
```

Comment:
Why need a constant-defined delay (In general it's sort of experience value) here to resume blacklist? Is it possible to enable blacklist back in more accurate place?

Reply:
Actually it is just existing thread that has been moved up.  Any reason why it is moved earlier now?

Reply:
Because the `MoveUpdatesToDisk` may take longer time than before. We can detach this function which has a sleep to finish it earlier